### DISCUSSION TITLE
How to seed the database?

### DISCUSSION THREAD

**Discussion by alanmatiasdev** [tada5hi/typeorm-extension#192] [Q&A] [answered]
Since upgrading typeorm to version 0.3, I'm looking for an alternative to seed the database. I saw in the README of the lib, which I can use for this purpose, but I didn't find an example of how to do it. Is it possible to do this with this lib?

**Comment by tada5hi** [comment, marked as answer]
Hey @bralandealmeida,
the seeding functionality is very minimalistic at the moment and poor documented, but i will enhance it capabilities and documentation.

Greetings
Peter

**Comment by tada5hi** [comment]
Hey @bralandealmeida,
the typeorm-extension version [v2.1.0](https://github.com/tada5hi/typeorm-extension/releases/tag/v2.1.0) should provide the seeding functionality to a certain extent by now.
Please let me know, if something is still missing in your eyes or can be improved.

Greetings 
Peter

**Reply by alanmatiasdev** [reply]
> Hey @bralandealmeida, the typeorm-extension version [v2.1.0](https://github.com/tada5hi/typeorm-extension/releases/tag/v2.1.0) should provide the seeding functionality to a certain extent by now. Please let me know, if something is still missing in your eyes or can be improved.
> 
> Greetings Peter

Yes. Worked perfectly! Thanks for the update and the lib.

**Reply by tada5hi** [reply]
Your welcome 🤗, glad you like it.

**Comment by zlavoie** [comment]
@tada5hi does the seeding functionality work for items like monodb? If so, is it possible to provide an example? I keep getting EntityMetadataNotFoundError when using mongodb but not when using postgresql

**Reply by tada5hi** [reply]
i have not implemented support for mongodb yet :sweat_smile: , as there has been no demand for it. But I will take care of it and also support mongodb :v: :relaxed: .

**Reply by tada5hi** [reply]
@zlavoie can you verify if v2.6.2 is working for you ?
